dev/ColorPicker/APITests/ColorPickerTests.cs

```diff
@@ -232,6 +246,37 @@ public void ValidateHueRange()
             IdleSynchronizer.Wait();
         }
 
+        [TestMethod]
+        public void ValidateFractionalWidthDoesNotCrash()
+        {
+            ManualResetEvent spectrumLoadedEvent = new ManualResetEvent(false);
+
+            RunOnUIThread.Execute(() =>
+            {
+                ColorSpectrum colorSpectrum = new ColorSpectrum();
+                colorSpectrum.Width = 300.75;
```

Comment:
Do we need to turn off layout rounding or anything? Will this get rounded to 301 and hence avoid the buggy behavior?

Reply:
Layout rounding affects how we get displayed, but not underlying values - the bug comes when we read and only sometimes round ActualWidth/ActualHeight, which return Width/Height when they're explicitly set.  I confirmed that this hits the crash without my fix.